openhub/apps.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class OpenhubConfig(AppConfig):
```

Comment:
`OpenHub*` ?

Reply:
@jayvdb, Actually the name of the app is `openhub` and here it automatically converts the first letter to uppercase.